pkg/datapath/linux/config/config_test.go

```diff
@@ -382,32 +424,38 @@ func TestPreferredIPv6Address(t *testing.T) {
 }
 
 func TestNewHeaderfileWriter(t *testing.T) {
-	testutils.PrivilegedTest(t)
```

Comment:
`testutils.PrivilegedTest(t)` is still necessary here.

Reply:
Creating a namespace is also a privileged operation. :slightly_smiling_face: Try running the tests without sudo, it'll fail.